team-norms/delivery_healthcheck.md

```diff
@@ -0,0 +1,17 @@
+# Delivery Healthcheck
+
+It is the responsibility of the lead on each delivery team to provide an update on the health of their delivery in each week's BizOps showcase.
+
+A Delivery Healthcheck should answer:
+
+## Current iteration goal
+ - When was your current iteration started?
+ - When is your next showcase?
+ - Are you on track to meet customer expectations?
+
+## Next iteration goal
+ - What is the goal for your next iteration?
+
+## Retrospective
+ - When was your last retrospective?
+ - What were your last retrospective action points?
```

Comment:
more a general question - what if you've not had a retro since the previous one. Repeat the same action points or just say like "We've not had one since" ?

Reply:
Yes, I'd expect exactly that. Of course, if you come to more than a couple of BizOps showcases without having had a more recent retrospective, questions may be asked as to your decision to eschew more frequent retros